storageClient/api.go

```diff
@@ -52,6 +52,9 @@ type DescribeGroupRequest struct {
 	GroupID uuid.UUID `json:"group_id"`
 }
 
+// GetGroupMembersRequest wraps the group id needed to look up group members
+type GetGroupMembersRequest DescribeGroupRequest
```

Comment:
now that I think about it, for both this and the api spec, this should be thought of as a list endpoint, because it returns more than one resource, Get/Describe are the verbs to use for fetching a single resource.

Reply:
ListGroupMembersRequest